src/compas_viewer/components/slider.py

```diff
@@ -2,176 +2,79 @@
-    value : int, optional
-        The initial value of the slider.
-        Defaults to 0.
-    min_value : int, optional
-        The minimum value of the sliding range.
-        Defaults to 0.
-    max_value : int, optional
-        The maximum value of the sliding range.
-        Defaults to 100.
-    step : int, optional
-        Size of value increments.
-        Defaults to 1.
-    title : str, optional
-        Title label.
-    annotation : str, optional
-        Value annotation label.
-    interval : int, optional
-        The tick interval size.
-        Defaults to 1.
-    bgcolor : :class:`compas.colors.Color`, optional
-        Background color of the box containing the slider.
-    stretch : int, optional
-        Stretch factor of the slider in the grid layout.
-        Defaults to 0.
-    kwargs : dict, optional
-        Additional keyword arguments for the action.
-
-    Attributes
-    ----------
-    action : callable
-        Action associated with the button click event.
-    slider : QtWidgets.QSlider
-        The actual slider widget.
-    value : float
-        The current value of the slider.
-    STYLE : str
-        Stylesheet for the visual appearance of the groove and handle of the slider.
-
-    See Also
-    --------
-    :class:`compas_viewer.layout.SidedockLayout`
-
-    References
-    ----------
-    :PySide6:`PySide6/QtWidgets/QSlider`
-
-    """
-
-    STYLE = """
-    QSlider::groove::horizontal {
-        border: 1px solid #cccccc;
-        background-color: #eeeeee;
-        height: 4px;
-    }
-    QSlider::handle:horizontal {
-        background-color: #ffffff;
-        border: 1px solid #cccccc;
-        border-radius: 6px;
-        height: 12px;
-        width: 12px;
-        margin: -6px 0;
-    }
-    """
 
+class Slider(QWidget):
     def __init__(
         self,
-        action: Callable,
-        value: int = 0,
-        min_value: int = 0,
-        max_value: int = 100,
-        step: int = 1,
-        title: Optional[str] = None,
-        annotation: Optional[str] = None,
-        interval: int = 1,
-        bgcolor: Optional[Color] = None,
-        stretch: int = 0,
-        **kwargs,
+        title: str = "Slider",
+        horizontal: Optional[bool] = True,
+        min_val: float = 0,
+        max_val: float = 100,
+        action: Callable = None,
+        tick_interval: Optional[float] = None,
+        starting_val: Optional[float] = None,
     ):
+        """
+        Initialize the slider with labels to display its state.
+
+        :param orientation: Qt.Horizontal or Qt.Vertical for slider orientation.
+        :param min_val: The minimum value of the slider.
+        :param max_val: The maximum value of the slider.
+        :param change_callback: A function to call when the slider value changes.
+        :param tick_interval: The interval between ticks displayed on the slider (optional).
+        :param starting_val: The starting value of the slider (optional).
+        """
```

Comment:
This is not the doc string format we use, could you align this? It needs to contain at least these sections
```
    Description...

    Parameters
    ----------
    ...
   
    Attributes
    ----------
    ...
```